config/config.go

```diff
@@ -0,0 +1,111 @@
+package config
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"os"
+	"reflect"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+)
+
+const (
+	defaultDaemonEndpoint   = "localhost:9000"
+	defaultKrakenWsEndpoint = "ws.kraken.com"
+	defaultBaseAsset        = "5ac9f65c0efcc4775e0baec4ec03abdde22473cd3cf33c0419ca290e0751b225"
+	defaultQuoteAsset       = "d73f5cd0954c1bf325f85d7a7ff43a6eb3ea3b516fd57064b85306d43bc1c9ff"
+	defaultKrakenTicker     = "XBT/USD"
+	defaultInterval         = 10
+)
+
+type Config struct {
+	DaemonEndpoint   string   `json:"daemon_endpoint,required"`
+	DaemonMacaroon   string   `json:"daemon_macaroon"`
+	KrakenWsEndpoint string   `json:"kraken_ws_endpoint,required"`
+	Markets          []Market `json:"markets,required"`
+}
+
+type Market struct {
```

Comment:
I don't know if it's the right place to put the Market struct definition. Maybe a `types.go` file makes more sense